.hlint-universum.yaml

```diff
@@ -0,0 +1,1050 @@
+############################################################################
+## Universum
+############################################################################
+
+# There's no 'head' in Universum
+- ignore: {name: "Use head"}
+
+# We have 'whenJust' for this
+- ignore: {name: "Use Foldable.forM_"}
+
+- warn: {lhs: Data.Text.pack, rhs: Universum.toText}
+- warn: {lhs: Data.Text.unpack, rhs: Universum.toString}
+
+- warn: {lhs: Data.Text.Lazy.pack, rhs: Universum.toLText}
+- warn: {lhs: Data.Text.Lazy.unpack, rhs: Universum.toString}
+- warn: {lhs: Data.Text.Lazy.toStrict, rhs: Universum.toText}
+- warn: {lhs: Data.Text.Lazy.fromStrict, rhs: Universum.toLText}
+
+- warn: {lhs: Data.Text.pack (show x), rhs: Universum.show x}
+- warn: {lhs: Data.Text.Lazy.pack (show x), rhs: Universum.show x}
+
+- warn: {lhs: Control.Exception.evaluate, rhs: evaluateWHNF}
+- warn: {lhs: Control.Exception.evaluate (force x), rhs: evaluateNF x}
+- warn: {lhs: Control.Exception.evaluate (x `deepseq` ()), rhs: evaluateNF_ x}
+- warn: {lhs: void (evaluateWHNF x), rhs: evaluateWHNF_ x}
+- warn: {lhs: void (evaluateNF x), rhs: evaluateNF_ x}
+
+## Monad.Trans
+
+- warn: {lhs: flip runReaderT , rhs: usingReaderT }
+- warn: {lhs: flip runReader  , rhs: usingReader }
+- warn: {lhs: flip runStateT  , rhs: usingStateT }
+- warn: {lhs: flip runState   , rhs: usingState }
+- hint: {lhs: fst <$> usingStateT s st, rhs: evaluatingStateT s st}
+- hint: {lhs: fst (usingState s st), rhs: evaluatingState s st}
+- hint: {lhs: snd <$> usingStateT s st, rhs: executingStateT s st}
+- hint: {lhs: snd (usingState s st), rhs: executingState s st}
+
+## Containers
```

Comment:
`keys`, `elems`, `toPairs` are non-standard as well, don't wanna force to use them (but less than the previous block).

Reply:
I disagree on this one, because unlike `{using,evaluating,executing}{Reader,State}{,T}` these functions provide a consistent interface to multiple containers instead of increasing cognitive burden for remembering names.